### PR TITLE
Replace practise desc with advisement class

### LINKED ISSUE #2133: Best practices should use .advicement class
Best practice boxes should use the W3C’s advicement css class. We currently use out own custom style.

### PR DESCRIPTION
**Pull request by isunitha98selvan**
The wiki needs to be modified for best practises from
`<p class="practicedesc">
`to 
`<p class="advisement">`

closes #2133

### CODE REVIEW

**Comment by saschanaz** [comment]
This doesn't close #2133. The change itself is good as the generator never uses `.practicedesc`.

I think we want to replace `.practicebox` with `.advisement` and `.practicetitle` with `.marker`.

**Comment by isunitha98selvan** [comment]
@saschanaz changes have been made

File: assets/bp.css [hunk 1]
```diff
@@ -6,16 +6,6 @@ span.practicelab {
...
-  color: #005A9C;
-  text-decoration: underline;
-  margin-bottom: 1em;
-}
-
-.practicebox {
-    background: black;
-}
-
 @media (max-width: 767px) {
```
Comment (marcoscaceres) [inline assets/bp.css]: Probably don't need this anymore, as practicetitle won't be used.

Comment (isunitha98selvan) [reply assets/bp.css]: Removed

File: assets/bp.css [hunk 2]
```diff
@@ -4,29 +4,4 @@ span.practicelab {
...
-.practicebox {
-    background: black;
-}
-
-@media (max-width: 767px) {
-  span.practicetitle {
-    display: block;
-  }
-}
+}
```
Comment (saschanaz) [inline assets/bp.css]: nit: a newline at the end of the file

Comment (isunitha98selvan) [reply assets/bp.css]: has been added

**Comment by saschanaz** [comment]
Looks like I was wrong, the `practicebox` should be kept and `.practicelab` should be replaced with `.advisement`. Also, the tr-design document currently defines `.advisement` only valid in `<strong>`/`<p>`/`<div>` while we only supports `<span>`. Maybe we should convert `<span>`s into those valid elements.

**Comment by marcoscaceres** [comment]
Using `strong` for `span`s would be good.

**Comment by marcoscaceres** [comment]
... but honestly, I'm not sure why this isn't just handled in the issues-notes.js module. This is just another kind of box with a different color.

**Pull request closed without merging**